jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/websocket/WebSocketDiagnosticsCollector.java

```diff
@@ -32,6 +37,78 @@ public void completeDiagnostic(Diagnostic diagnostic) {
 
     @Override
     public void collectDiagnostics(ICompilationUnit unit, List<Diagnostic> diagnostics) {
-
+    	if (unit != null) {
+    		try {
+    			IType[] alltypes = unit.getAllTypes(); 
+    			for (IType type : alltypes) {
+    				/* @PathParam Value Mismatch Warning */
+    				List<String> endpointPathVars = findAndProcessEndpointURI(type);
+    				if (endpointPathVars == null) {
+    					continue;
+    				}
+    				IMethod[] typeMethods = type.getMethods();
+    				for (IMethod method : typeMethods) {
+    					ILocalVariable[] methodParams = method.getParameters();
+    					for (ILocalVariable param : methodParams) {
+    						IAnnotation[] paramAnnotations = param.getAnnotations();
+    	    				for (IAnnotation annotation : paramAnnotations) {
```

Comment:
Insignificant issue but formatting is off in this part and its corresponding closing statement